app-forensics/yara/yara-4.2.0-r1.ebuild

```diff
@@ -51,5 +53,5 @@ src_install() {
 
 	# TODO: Allow tests to work against dyn. lib rather than building
 	# statically just for tests.
-	find "${ED}" -name '*.a' -delete || die
+	find "${ED}" -name '*.a' -o -name '*.la' -delete || die
```

Comment:
FWIW: no need for revbump for test fix if it's purely a test fix (as opposed to fixing some runtime issue exposed by tests), but for .la files, debatable. I'd say yes given in theory they can break stuff.